src/stores/sdkConfig/SDKConfigStore.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  EVM,
+  Solana,
+  UTXO,
+  Sui,
+  SDKConfig,
+  createConfig as lifiSdkCreateConfig,
+  config as lifiSdkConfig,
+} from '@lifi/sdk';
+import { publicRPCList } from 'src/const/rpcList';
+import getApiUrl from 'src/utils/getApiUrl';
+import { create } from 'zustand';
+
+export enum ConfigType {
+  Default = 'default',
+  Zap = 'zap',
+}
+
+interface SDKConfigState {
+  configType: ConfigType;
+  partialConfigMap: Record<ConfigType, Partial<SDKConfig>>;
+  config: SDKConfig;
+
+  initSdk: () => void;
+  setConfigType: (type: ConfigType) => void;
+}
+
+const integrator = process.env.NEXT_PUBLIC_WIDGET_INTEGRATOR;
+
+const baseConfig: SDKConfig = {
+  apiKey: process.env.NEXT_PUBLIC_LIFI_API_KEY,
+  apiUrl: getApiUrl(),
+  providers: [EVM(), Solana(), UTXO(), Sui()],
+  integrator,
+  rpcUrls: {
+    ...JSON.parse(process.env.NEXT_PUBLIC_CUSTOM_RPCS),
+    ...publicRPCList,
+  },
+  preloadChains: true,
+};
+
+const zapConfig: Partial<SDKConfig> = {
+  providers: [EVM(), Solana(), UTXO()],
+};
```

Comment:
Zap can also have Sui, it does not matter in this context. So probably should be removed.